src/pylero/document.py

```diff
@@ -316,6 +316,30 @@ def query(
             lst_doc = [Document(suds_object=doc) for doc in docs]
             return lst_doc
 
+    @classmethod
+    def export_pdf(cls, uri, exportPdfProperties):
```

Comment:
add a output dir path option for the function, then return the exported file path with:
path dir + the doc name + .pdf

Reply:
updated